src/qt/optionsmodel.h

```diff
@@ -116,6 +118,7 @@ class OptionsModel : public QAbstractListModel
     bool m_use_embedded_monospaced_font;
     bool fCoinControlFeatures;
     bool m_sub_fee_from_amount;
+    bool fEnablePSBTControls;
```

Comment:
Any specific reason for using the antiquated `fEnablePSBTControls` convention here, or was this an oversight? (I see you tried to use `m_` elsewhere)

Reply:
Changed to `m_enable_psbt_controls`.